.travis.yml

```diff
@@ -1,14 +1,22 @@
 language: scala
+sudo: required
+dist: trusty
 env:
   global:
-    _JAVA_OPTIONS="-Xmx1500m -XX:MaxPermSize=512m -Dakka.test.timefactor=3"
+    - _JAVA_OPTIONS="-Xmx1500m -XX:MaxPermSize=512m -Dakka.test.timefactor=3"
+    - SPARK_HOME=/tmp/spark-1.6.2-bin-hadoop2.6
 scala:
    - 2.10.6
    - 2.11.8
 jdk:
    - oraclejdk8
    - oraclejdk7
+before_install:
+  - sudo apt-get -qq update
+  - sudo apt-get -y install python3 python3-pip
```

Comment:
No, both Python2 and 3 are supported. But the image already contains python2 and pip2. Note that `ci/install-python-dependencies.sh` installs pyhocon for both. There are tests for both in `SubprocessSpec.scala`. 

The reason for changing the `dist` in the Travis file is also related to this, the older default Travis image is Ubuntu 12.04 for which installing Python3 and pip3 is awkward: http://askubuntu.com/questions/412178/how-to-install-pip-for-python-3-in-ubuntu-12-04-lts